app/assets/javascripts/miq_tree.js

```diff
@@ -217,16 +217,6 @@ function miqOnClickHostNet(id) {
   }
 }
 
-// OnClick handler for Timeline Tree
-function miqOnClickTimelineSelection(id) {
-  var allIds = id.split('rep-');
-  // Valid id should be something like xx-first_xx-second_rep-1r215, so it should return two 'ids' if split by rep- identifier
-  if (allIds.length === 2) {
-    var rep_id = allIds[allIds.length - 1];
-    miqJqueryRequest(ManageIQ.tree.clickUrl + '?id=' + rep_id, {beforeSend: true, complete: true});
-  }
-}
-
```

Comment:
I still see `miqOnClickTimelineSelection` being mentioned once in `miq_tree.js`.